### DISCUSSION TITLE
Online Accounts

### DISCUSSION THREAD

**Discussion by TheNetRanger** [elementary/triage#344] [Ideas]
Please add a simple method of adding accounts such as Gmail, iCloud and Outlook. I know its possible to add the Gnome stuff to do this, but not user friendly for a beginner, would be much nicer if just built in as an option

**Comment by fayjies** [comment]
I remember discussing this with @danirabbit once. The Gnome way is not very suitable for elementary os regarding online accounts. However, I believe, elementary OS should start addressing this issue somehow, as it is a proper replacement for Windows and Mac OS,  I would expect Yahoo, AOL, exchange, calendar integration, Gmail, etc in any modern OS.
